tests/sentry/api/endpoints/notifications/test_notification_actions_index.py

```diff
@@ -128,10 +146,11 @@ def test_get_with_queries(self, mock_trigger_types):
         }
 
         for data in query_data.values():
+            assert isinstance(data, MutableMapping), "query_data's values need to be indexable ;-;"
```

Comment:
this assert shouldn't be needed -- it's unhappy because `query_data` is being inferred as `dict[str, object]` -- probably needs an annotation and/or a TypedDict

looks like it's:

```python
class _Query(TypedDict, total=False):
    triggerType: str
    project: int

class _QueryResult(TypedDict):
    query: _Query
    result: set[NotificationAction]

query_data: dict[str, _QueryResult] = { ... }